Provisioning/README.md

```diff
@@ -7,7 +7,7 @@ This is an alternative to using the in-app provisioning tool.
 
 - A subscription to [Microsoft Azure](https://azure.microsoft.com/free/).
 - [A basic bot built using Composer](https://aka.ms/composer-create-first-bot).
-- Latest version of the [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli).
+- [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli) version 2.36.0 or earlier.
```

Comment:
This should probably just be the specific version `2.36.0`, or at least a specific range of versions. I don't know what the earliest version is of the Azure CLI that works with Composer.

Reply:
Done.